### PR TITLE
Bringing createTimeGate into the main 

### PR DESCRIPTION
**Pull request by lrochette**
I need the timeGate for our current user
I added documentation for the timeGate

I also had some issues when promoting for the second time the same version but it seems you added the code to delete old Acls already

Also let me know how I can make my code more in line with the "default" template

Thanks

### CODE REVIEW

File: src/main/resources/project/ec_setup.pl [hunk 1]
```diff
@@ -75,18 +75,18 @@
...
 );
 
-#my %createScheduleToTriggerGate = (
-#  label       => "EF-Utilities - createScheduleToTriggerGate",
-#  procedure   => "createScheduleToTriggerGate",
-#  description => "Create a schedule that trigger a gate at a particular time and date",
-#  category    => "Deploy"
-#);
+my %createScheduleToTriggerGate = (
+  label       => "EF-Utilities - createScheduleToTriggerGate",
```
Comment (vpalaiya) [inline src/main/resources/project/ec_setup.pl]: Should the label be in camel case?

It seems that procedure name is displayed in the UI. Should it follow same pattern as "Decommission Environments", that is, no camel case?

Also help.xml has name as Create Time Gate. Shouldn't what's displayed in UI be consistent with it?

File: src/main/resources/project/ec_setup.pl [hunk 2]
```diff
@@ -75,18 +75,18 @@
...
+  procedure   => "createScheduleToTriggerGate",
+  description => "Create a schedule that trigger a gate at a particular time and date",
```
Comment (vpalaiya) [inline src/main/resources/project/ec_setup.pl]: minor feedback - plural for "triggers", and "date and time"

File: src/main/resources/project/ec_setup.pl (hunk 1 above)
Comment (justnoxx) [inline src/main/resources/project/ec_setup.pl]: In perl world there is no any limitations to naming, but, we can use camelCase or underscore_naming if it's reasonable, for example, if project written in camelCase, we can use camelCase, if underscore, we can use underscore. Indentation and code formatting are much more important.

Comment (bva) [inline src/main/resources/project/ec_setup.pl]: Label for procedure should contain short description of what it's doing. Without 'EC-' (or 'EF-' in this case) prefix. For this case, i think, 'Flow Utilities - Create Schedule To Trigger Gate' or something similar would be enough.

File: src/main/resources/project/ec_setup.pl [hunk 3]
```diff
@@ -75,18 +75,18 @@
 
 # Data that drives the create step picker registration for this plugin.
 my %decommissionEnvironments = (
-	label     => "EF-Utilities - Decommission Environments",
-	procedure => "Decommission Environments",
+	label       => "EF-Utilities - Decommission Environments",
```
Comment (bva) [inline src/main/resources/project/ec_setup.pl]: See my previous note regarding naming.

File: src/main/resources/project/ec_setup.pl (hunk 1 above)
Comment (lrochette) [inline src/main/resources/project/ec_setup.pl]: fixed

File: src/main/resources/project/ec_setup.pl (hunk 2 above)
Comment (lrochette) [inline src/main/resources/project/ec_setup.pl]: not plural but 3rd person. Both fixed

**Comment by vpalaiya** [comment]
vitaliy, please confirm what's remaining to be done to finish this pull request

File: pages/help.xml [hunk 4]
```diff
@@ -50,6 +50,51 @@
                     </tbody>
                 </table>
             </div>
+
+            <div id="Create Time Gate">
+                <a name="Create Time Gate" id="Create Time Gate"></a>
+
```
Comment (bva) [inline pages/help.xml:56]: Should be 'Create Schedule To Trigger Gate', i think

File: src/main/resources/project/ec_setup.pl (hunk 3 above)
Comment (bva) [inline src/main/resources/project/ec_setup.pl]: Better 'Flow Utilities - Decommission Environments', same for 'Create Schedule To Trigger Gate' -> 'Flow Utilities - Create Schedule To Trigger Gate'

File: pages/help.xml (hunk 4 above)
Comment (lrochette) [inline pages/help.xml:56]: fixed

Comment (bva) [inline pages/help.xml:56]: Laurent, I still see 'Create Time Gate' in docs, did you pushed last commit?

File: src/main/resources/project/ec_setup.pl (hunk 3 above)
Comment (lrochette) [inline src/main/resources/project/ec_setup.pl]: Fixed

**Pull request merged**